<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Scheduler</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addProcessButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Scheduling</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="processTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>960</width>
      <height>250</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>PID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Arrival</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Burst</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Finish</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Waiting</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Turnaround</string>
     </property>
    </column>
   </widget>
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>960</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="comparisonChart" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>960</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
